### PR TITLE
Update roll data and active effect wikis for 3.x

### LINKED ISSUE #2755: Spell save DC bonus is missing from the AE guide
`system.bonuses.spell.dc` (+number/formula)

### PR DESCRIPTION
**Pull request by krbz999**
Closes #2755.

### CODE REVIEW

File: wiki/Roll-Formulas.md [hunk 1]
```diff
@@ -52,10 +48,6 @@
 
 #### Armor Class
 
-`@attributes.ac.calc` - Calculation mode that its used for determining `attributes.ac.base`
-
-`@attributes.ac.formula` - Custom formula that will be used to determine `attributes.ac.base` if `calc` is set to `custom`
```
Comment (Fyorl) [inline wiki/Roll-Formulas.md:57]: Why are these removed?

File: wiki/Roll-Formulas.md [hunk 2]
```diff
@@ -79,201 +71,163 @@
 
 `@attributes.attunement.max` - Maximum number of attunement slots
 
+#### Concentration
+
+`@attributes.concentration.limit` - Maximimum number of effects the actor can sustain concentration on.
+
+`@attributes.concentration.roll.min` and `.max` - The minimum and maximum values of the d20 for concentration-related saving throws.
```
Comment (Fyorl) [inline wiki/Roll-Formulas.md:78]: These don't do anything currently, do they? Probably shouldn't mention them on the wiki in that case.

File: wiki/Roll-Formulas.md [hunk 3]
```diff
@@ -79,201 +71,163 @@
...
-
-`@bonuses.rsak` - Ranged spell attack
-
-`@bonuses.rwak` - Ranged weapon attack
-
-***Note:** Replace the `*` in the following formulas with the four-letter code from above.*
-
-`@bonuses.*.attack` - Global bonus to attack rolls
-
-`@bonuses.*.damage` - Global bonus to damage rolls
```
Comment (Fyorl) [inline wiki/Roll-Formulas.md:181]: Why are we removing these? Did they get moved somewhere else?

File: wiki/Roll-Formulas.md [hunk 4]
```diff
@@ -315,40 +269,46 @@
 
 ***Note:** Replace the `*` in the following formulas with the three-letter code from above.*
 
-`@skills.*.ability` - Three letter code for the ability associated with this skill by default
```
Comment (Fyorl) [inline wiki/Roll-Formulas.md:318]: Unclear why this is and the bonuses were removed.

**Review by Fyorl — CHANGES_REQUESTED** [review]

File: wiki/Roll-Formulas.md (hunk 1 above)
Comment (krbz999) [reply wiki/Roll-Formulas.md:57]: They can't be used. Unsure why they were ever included; you can't construct a roll formula with the word "default", and we don't (off the top of my head) double-evaluate roll data to allow for use of `ac.formula`.

File: wiki/Roll-Formulas.md (hunk 3 above)
Comment (krbz999) [reply wiki/Roll-Formulas.md:181]: Same as before, they can't be used as roll data. But they are in the AE wiki.

File: wiki/Roll-Formulas.md (hunk 2 above)
Comment (krbz999) [reply wiki/Roll-Formulas.md:78]: Agree. Replacing with `.save`, which is a number (added during data prep).

File: wiki/Roll-Formulas.md (hunk 3 above)
Comment (Fyorl) [reply wiki/Roll-Formulas.md:181]: Hmm, I see. Well it's probably fine so long as they're in the AE wiki, but some of them, like `@bonuses.mwak` certainly could be used in roll data so long as they don't also contain roll data references themselves. I suppose I don't really know when you would want to do that though.

File: wiki/Roll-Formulas.md (hunk 2 above)
Comment (Fyorl) [reply wiki/Roll-Formulas.md]: `roll.min` and `roll.max` appear in a few other places in this changeset as well. I don't know if perhaps I'm getting confused with the overlap with AEs maybe, but I don't think those fields can even be configured in the UI (yet) so I'm not sure about including them in the wiki at this stage.

Comment (krbz999) [reply wiki/Roll-Formulas.md]: Just me being focused on conc. They should not be there if they are not functioning yet.

File: wiki/Roll-Formulas.md (hunk 3 above)
Comment (krbz999) [reply wiki/Roll-Formulas.md:181]: I think excluding properties that can lead to issues in that way is a good rule of thumb to use (else we could or should technically include `@name` if you name your character "5").

**Review by Fyorl — APPROVED** [review]
Need to wait to merge this until 3.1 is released.
